accesscontrol/oauth2.go

```diff
@@ -26,12 +26,12 @@ func NewOAuth2Callback(oauth2Client oauth2.AcClient) (*OAuth2Callback, error) {
 // Validate implements the AccessControl interface
 func (oa *OAuth2Callback) Validate(req *http.Request) error {
 	if req.Method != http.MethodGet {
-		return errors.Oauth2.Messagef("wrong method: %s", req.Method)
+		return errors.AccessControl.Messagef("wrong method: %s", req.Method)
```

Comment:
To have a bad request error this must map to `errors.Oauth2`. Otherwise `errors.Oauth2` can be removed since there is no reference at all.